wordCount.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+)
+
+func wordCount(s string) map[string]int {
+	ss := strings.Split(s, " ")
+	fmt.Println(ss)
+	visited := make(map[string]bool)
+	result := make(map[string]int)
+	for _, item := range ss {
+		if visited[item] == true || item==""{
+			continue
+		} else {
+			visited[item] = true
+			c:=countSubString(item,ss)
```

Comment:
It's not effective in terms of performance to run for cycle inside the for cycle. You don't have to use nesting cycles to implement word count logic.